runtests.py

```diff
@@ -10,7 +10,7 @@
 # Unset ENV variables so that tests use default values.
 for env_var in ['FHIR_URL', 'DJANGO_MEDICARE_LOGIN_URI',
                 'DJANGO_SLS_USERINFO_ENDPOINT', 'DJANGO_SLS_TOKEN_ENDPOINT',
-                'DJANGO_FHIR_CERTSTORE']:
+                'DJANGO_FHIR_CERTSTORE', 'DATABASES_CUSTOM']:
```

Comment:
When running the tests in a Docker container, it was using the Postgres database. However, Travis CI and our integration tests are using SQLite3. This caused a few of the IntegrityError type tests to fail, since the message text was different between the two databases. Unsetting `DATABASES_CUSTOM` enables the tests to use SQLite3 for local (Dockerized) development.